modules/tensor_mechanics/src/kernels/StressDivergenceRSphericalTensors.C

```diff
@@ -62,9 +62,6 @@ StressDivergenceRSphericalTensors::computeQpOffDiagJacobian(unsigned int jvar)
     if (jvar == _disp_var[i])
       return calculateJacobian(_component, i);
 
-  if (_temp_coupled && jvar == _temp_var)
-    return 0.0;
-
   return 0.0;
```

Comment:
Do we have a plan to implement the OffDiagJacobian terms for Spherical coordinate system?

Reply:
No, but I see why you would be interested in that :-)